pages/info/about.md

```diff
@@ -6,65 +6,105 @@ toc: false
+A Unix operating system is broken into two primary components, the kernel space, and the user space. The Kernel supports the user space by interfacing with the hardware, providing core system features and creating the software compatibility layers for the user space. The user space on the other hand is the environment that most people are most familiar with interfacing with. It is where applications, libraries and system services run.
+
+Containers are shifting the emphasis away from the runtime environment by commoditizing the user space into swappable components. This means that the entire user space portion of a Linux operating system, including programs, custom configurations, and environment can be interchanged at runtime. Singularity emphasis and simplifies the distribution vector of containers to be that of a single, verifiable file.
 
-Some of the benefits of distributing software in this fashion include being able to provide a configuration free implementation which can easily be replicated to distribute a software or work flow. The user does not need to worry about managing potentially complicated dependency chains nor worry about configuration nuances.
+Software developers can now build their stack onto whatever operating system base fits their needs best, and create distributable runtime encapsulated environments and the users never have to worry about dependencies, requirements, or anything else from the user space.
 
-The drawbacks are numerous, but tend to focus on several major areas: container overhead, container technology and architecture concerns (e.g., privilege escalation and network/file system access), and work flow compatibility.
+Singularity provides the functionality of a virtual machine, without the heavyweight implementation and performance costs of emulation and redundancy!
 
 ### The Singularity Solution
-Singularity containers are purpose built and can include a simple binary and library stack or a complicated work flow that includes both network and file system access (or anything in between). The Singularity container images are then completely portable to any binary compatible version of Linux with the only dependency being Singularity running on the target system.
+Singularity has two primary roles:
+
+1. **Container Image Management:** Singularity supports building different container image formats from scratch using your choice of Linux distribution bases or leveraging other container formats (e.g. Docker Hub). Container formats supported are the default compressed immutable (read only) image files, writable raw file system based images, and sandboxes (*chroot* style directories). 
```

Comment:
Would we really call this management? When I think management I think of Singularity Registry (eg, an actual manager to organizer and control images). Singularity is the maker of the containers, but not really the organizer. This paragraph is getting more at Container Image Formats. But I see you want it to be a "role." Maybe just Container Image Generator? THE BEST Container Image ? lol

Reply:
Strong Container Base

Reply:
Image generator works too! I'm fine with it :)

Reply:
okay I like that too, let's change to that.